examples/dt/uni01alpha/control-plane/service-values.yaml

```diff
@@ -0,0 +1,135 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  cinderVolumes:
+    replicas: 1
```

Comment:
Replicas needs to be nested under the lvm-iscsi stanza (below L13).

For background, while this CR deploys a single cinder-volume backend, multiple backends are supported and each one has a replicas field.

Reply:
Thank you. Let me make the changes in the next commit.